<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMultiVolumeRenderingModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMultiVolumeRenderingModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonRendering">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rendering</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="labelRenderingMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxRenderingMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Linked Multi-Volume Rendering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Independent Multi-Volume Rendering</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelGPUMemSize">
        <property name="text">
         <string>GPU Memory Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxGPUMemSize">
        <item>
         <property name="text">
          <string>512 MB (default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0 GB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5 GB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0 GB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelQualityControl">
        <property name="text">
         <string>Quality Control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxQualityControl">
        <item>
         <property name="text">
          <string>Adaptive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed Framerate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maximum Quality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelFps">
        <property name="text">
         <string>Interactive Speed</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="ctkSliderWidget" name="SliderWidgetFps">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>30.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
        <property name="suffix">
         <string> fps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxDisplayNode"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDisplayNode">
        <property name="text">
         <string>Display Node</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonVolumes">
     <property name="text">
      <string>Volumes</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxLabelmapVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLabelMapVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxForegroundVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist>
          <string>vtkMRMLDiffusionTensorVolumeNode</string>
          <string>vtkMRMLDiffusionWeightedVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="ctkCheckBox" name="CheckBoxLabelmapVolume">
        <property name="text">
         <string>Labelmap</string>
        </property>
        <property name="indicatorIcon">
         <iconset resource="../../../Annotations/Resources/qSlicerAnnotationModule.qrc">
          <normaloff>:/Icons/AnnotationVisibility.png</normaloff>
          <normalon>:/Icons/AnnotationInvisible.png</normalon>:/Icons/AnnotationVisibility.png</iconset>
        </property>
        <property name="indicatorIconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ctkCheckBox" name="CheckBoxBgVolume">
        <property name="text">
         <string>Background</string>
        </property>
        <property name="indicatorIcon">
         <iconset resource="../../../Annotations/Resources/qSlicerAnnotationModule.qrc">
          <normaloff>:/Icons/AnnotationVisibility.png</normaloff>
          <normalon>:/Icons/AnnotationInvisible.png</normalon>:/Icons/AnnotationVisibility.png</iconset>
        </property>
        <property name="indicatorIconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxBackgroundVolume">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist>
          <string>vtkMRMLDiffusionWeightedVolumeNode</string>
          <string>vtkMRMLDiffusionTensorVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ctkCheckBox" name="CheckBoxFgVolume">
        <property name="text">
         <string>Foreground</string>
        </property>
        <property name="indicatorIcon">
         <iconset resource="../../../Annotations/Resources/qSlicerAnnotationModule.qrc">
          <normaloff>:/Icons/AnnotationVisibility.png</normaloff>
          <normalon>:/Icons/AnnotationInvisible.png</normalon>:/Icons/AnnotationVisibility.png</iconset>
        </property>
        <property name="indicatorIconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonBgPara">
     <property name="text">
      <string>Parameters (Bg Volume)</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLabel" name="labelROINodeBg">
        <property name="text">
         <string>ROI Node</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxROINodeBg"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxCropBg">
          <property name="text">
           <string>Crop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPropertyNodeBg">
        <property name="text">
         <string>Property</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxPropertyBg"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPresetsBg">
        <property name="text">
         <string>Preset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qSlicerPresetComboBox" name="PresetComboBoxBg"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBoxAdvancedBg">
        <property name="title">
         <string>Advanced</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidgetAdvancedBg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabTechniqueBg">
            <property name="accessibleName">
             <string/>
            </property>
            <attribute name="title">
             <string>Technique</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="labelTechniqueBg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Technique</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBoxTechniqueBg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Composite</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Maximum Intensity Projection (MIP)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Minimum Intensity Projection (MINIP)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabVolumePropertyBg">
            <attribute name="title">
             <string>Volume Property</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="ctkCheckablePushButton" name="CheckBoxSyncWithVolumesBg">
               <property name="text">
                <string>Synchronize with Volumes module</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="qMRMLVolumePropertyNodeWidget" name="MRMLVolumePropertyNodeWidgetBg"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabROIBg">
            <attribute name="title">
             <string>ROI</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="qMRMLAnnotationROIWidget" name="MRMLAnnotationROIWidgetBg"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonFgPara">
     <property name="text">
      <string>Parameters (Fg Volume)</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="labelROINodeFg">
        <property name="text">
         <string>ROI Node</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPropertyNodeFg">
        <property name="text">
         <string>Property</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxPropertyFg"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPresetsFg">
        <property name="text">
         <string>Preset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qSlicerPresetComboBox" name="PresetComboBoxFg"/>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxROINodeFg"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxCropFg">
          <property name="text">
           <string>Crop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBoxAdvancedFg">
        <property name="title">
         <string>Advanced</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidgetAdvancedFg">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabTechniqueFg">
            <attribute name="title">
             <string>Technique</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tabVolumePropertyFg">
            <attribute name="title">
             <string>Volume Property</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="qMRMLVolumePropertyNodeWidget" name="MRMLVolumePropertyNodeWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabROIFg">
            <attribute name="title">
             <string>ROI</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="CollapsibleButtonLabelmapPara">
     <property name="text">
      <string>Parameters (Labelmap)</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="labelROINodeLabelmap">
        <property name="text">
         <string>ROI Node</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxROINodeLabelmap"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxCropLabelmap">
          <property name="text">
           <string>Crop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLAnnotationROIWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLAnnotationROIWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumePropertyNodeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLVolumePropertyNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerPresetComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qSlicerPresetComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>QPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Annotations/Resources/qSlicerAnnotationModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerMultiVolumeRenderingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBoxBackgroundVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerMultiVolumeRenderingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBoxForegroundVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerMultiVolumeRenderingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBoxLabelmapVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
